client/my-sites/plans/main.jsx

```diff
@@ -31,7 +31,7 @@ const Plans = React.createClass( {
 	getDefaultProps() {
 		return {
 			intervalType: 'yearly',
-			displayJetpackPlans: true
+			displayJetpackPlans: false
```

Comment:
to be honest, I don't understand why we should be defaulting to anything. Keep in mind that plans/main.jsx is used to show every type of plans, not just for wpcom ones... So I don't think we should have any default that would mean that the experience can be broken just because someone forgot to add a param / didn't know better. I don't see why we should risk this just for adding a default here, default values would mean that something degrades gracefully, and this is not the case. IMO this param should be passed in all the cases, so why having a default value that could be problematic?

Also, I don't think we should split this into two different components. It would mean to have to maintain duplicated code over a flag difference. The thing here is that there's not real difference between wpcom and jetpack plans... they are the same data structure, rendered using the same components... Everything is the same, the only difference is that we are showing one or other set of plans.